host-bin/edit-chroot

```diff
@@ -594,6 +610,31 @@ for NAME in "$@"; do
             mv "$CHROOT" "$target"
         fi
     fi
+
+    # Copy the chroot if requested
```

Comment:
Actually, shouldn't this go above the MOVE section?  That way you can copy and move at the same time...